File: 1980-find-unique-binary-string/main.cpp
class Solution {
public:
    string findDifferentBinaryString(vector<string>& nums) {
        vector<char> ans;
        size_t n = nums.size();

        for (size_t i = 0; i < n; i++) {
            char curr = nums[i][i];
            ans.push_back(curr == '0' ? '1' : '0');
        }

        return string(ans.begin(), ans.end());
    }
};

File: 1980-find-unique-binary-string/main.go
func findDifferentBinaryString(nums []string) string {
    var ans []string
    n := len(nums)

    for i := range n {
        curr := string(nums[i][i])
        newLetter := "0"
        if curr == "0" {
            newLetter = "1"
        }

        ans = append(ans, newLetter)
    }

    return strings.Join(ans, "")
}
